Makefile

```diff
@@ -188,6 +188,12 @@ static/gen_manifests.go: $(shell find static/manifests -type f)
 .PHONY: generate-bindata
 generate-bindata: pkg/assets/zz_generated_offsets_$(TARGET_OS).go
 
+.PHONY: generate-APIClient
+
+# install with go get k8s.io/code-generator/cmd/client-gen@v0.22.2
```

Comment:
Would it make sense to include this command as a part of the target to avoid having the caller setup their environment properly?

Reply:
You can use `go env` for that:
```
generate-APIClient:
    go get k8s.io/code-generator/cmd/client-gen@v0.22.2
    $(go env GOPATH)/bin/client-gen --etc --etc
```